projects/core/build/tokens.ts

```diff
@@ -521,11 +521,11 @@ const aliases = {
         highlight: token(color.blue[700]),
       },
       color: {
-        value: token(color.construction[700]),
+        value: token(color.construction[500]),
         hover: token(color.construction[1000]),
         active: token(color.construction[1000]),
         selected: token(color.construction[700]),
-        disabled: token(color.construction[300]),
+        disabled: token(color.construction[700]),
```

Comment:
`--cds-alias-object-interaction-color-disabled`

Reply:
I'm not familiar with this change. Need some context. Possibly requested by another designer.

Reply:
This is an error on our part. It should still refer to construction 300 for light theme.

Reply:
This is for the light theme. Should this be mapped to 300 or 700?

Reply:
Looks like 300